src/stores/utils.js

```diff
@@ -56,7 +56,7 @@ async function getCrowdsaleAsset(contractName, stateProp, networkID) {
       ? setFlatFileContentToState(`./contracts/${stateProp}.sol`)
       : Promise.resolve()
   const whenBin =
-    stateProp === 'MintedCappedProxy' || stateProp === 'DutchProxy' || stateProp === 'ProxiesRegistry'
+    stateProp === 'MintedCappedProxy' || stateProp === 'DutchProxy'
```

Comment:
@fernandomg could you remind, why do we need to remove `ProxiesRegistry` from here?

Reply:
If you serve the app running `npm run start` it will try to find the `ProxiesRegistry` file and will fail, but instead of throwing a `404` error, it returns the `index.html` page content... and it's stored in the `CrowdsaleStore` as the `ProxiesRegistry`'s `bin`.

![image](https://user-images.githubusercontent.com/3315606/45368502-212de300-b5ba-11e8-808e-0e863f34689a.png)

While if you build the app (`npm run build`) and serve it from within the `/build` directory with `npx http-server` you'll be able to get what I assume we will get from netlify server (a `404` error). And the page won't continue.

![image](https://user-images.githubusercontent.com/3315606/45368609-5f2b0700-b5ba-11e8-8da9-5e66d04949c5.png)

---
As the app worked fine with the wrong `bin` for `ProxiesRegistry` I assumed that it's not required, besides we're not storing it in the `/contracts` directory.

If it will be required in the future, we can add it by that time. Plus we can properly handle the `404` and inform the user about the issue with a modal/toaster. I can create an issue for the last if you think it's necessary.

Reply:
@fernandomg you're right, we do not need to check `ProxiesRegistry` here. Thank you for the clarification. And yes, your proposal to properly handle `404` does make sense to me.